### PR TITLE
escape logo alt text and title

### PR DESCRIPTION
**Pull request by ErikPel**
### Description (*)

Added escapeHtmlattr to logo alt text so theres not XSS possibility by changing this text from admin panel
### Fixed Issues (if relevant)

1. magento/magento2#19269: Logo "alt" attribute not escaped; possible (self) XSS

### Manual testing scenarios (*)

1. Go to Content -> Design -> Configuration
2. Edit a theme (doesn't matter which) by clicking "Edit"
3. Scroll down to "Header" and add the following to the Logo Image Alt field
` " onload="javascript:alert('hello, world');`
4. Check whether you get javascript alert window or not

`Notice the leading "
### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @ErikPel. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by VladimirZaets — DISMISSED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3497](https://jira.corp.magento.com/browse/ENGCOM-3497) has been created to process this Pull Request

File: app/code/Magento/Theme/view/frontend/templates/html/header/logo.phtml [hunk 1]
```diff
@@ -18,8 +18,8 @@
     <a class="logo" href="<?= $block->getUrl('') ?>" title="<?= /* @escapeNotVerified */ $storeName ?>">
 <?php endif ?>
         <img src="<?= /* @escapeNotVerified */ $block->getLogoSrc() ?>"
-             title="<?= /* @escapeNotVerified */ $block->getLogoAlt() ?>"
-             alt="<?= /* @escapeNotVerified */ $block->getLogoAlt() ?>"
+             title="<?= /* @escapeNotVerified */ $block->escapeHtmlAttr($block->getLogoAlt()) ?>"
+             alt="<?= /* @escapeNotVerified */  $block->escapeHtmlAttr($block->getLogoAlt()) ?>"
```
Comment (orlangur) [inline app/code/Magento/Theme/view/frontend/templates/html/header/logo.phtml]: `escapeNotVerified` must be changed accordingly.

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/Theme/view/frontend/templates/html/header/logo.phtml (hunk 1 above)
Comment (ErikPel) [reply app/code/Magento/Theme/view/frontend/templates/html/header/logo.phtml]: removed them

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-3497](https://jira.corp.magento.com/browse/ENGCOM-3497) has been created to process this Pull Request

**Comment by sdzhepa** [comment]
Hello @ErikPel 

Could you please update your branch and resolve conflicts.

**Comment by VladimirZaets** [comment]
@ErikPel, I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-4033](https://jira.corp.magento.com/browse/ENGCOM-4033) has been created to process this Pull Request

**Comment by simonmaass** [comment]
any schedule when 2.3.1 will be released?

**Comment by wontroba666** [comment]
Hello, when 2.3.1 will be released? we need this fixes

**Comment by Bowens20832** [comment]
Any timeline? We need these bug fixes as well!

**Comment by DanielRuf** [comment]
If you need them you can already use the patch file: https://github.com/magento/magento2/pull/19270.patch

**Pull request merged**